### DISCUSSION TITLE
How to convert subdomain to subdirectory using sveltekit static adapter

### DISCUSSION THREAD

**Discussion by boradakash** [sveltejs/kit#10412] [Q&A]
My site is at blog.example.com and I want to work it on example.com/blog using cloudflare worker reverse proxy

Basically I want to convert anchror href from '/test' to '/blog/test' (means blog subdirectory).

I am using cloudflare worker HTMLRewriter to update anchor[href] and script[src] attribute and everything is being replaced except anchor[href]. It looks like when kit.start() method is being called it overwrites the anchor[href] to the actual conten of html 

so cloudflare worker HTMLRewriter response is being overwritten by actually code.

Any one know if I am doing anything wrong or is there any workaround for this problem that I am facing?

**Comment by ghostdevv** [comment]
Maybe you are looking for this? https://kit.svelte.dev/docs/configuration#paths

**Reply by boradakash** [reply]
Thanks, For me it was actually other issue because I was using cloudflare worker but I find a solution.
